Content.Server/Speech/EntitySystems/VulgarAccentSystem.cs

```diff
@@ -0,0 +1,67 @@
+using Content.Shared.Speech.Components;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Robust.Shared.Random;
+using Content.Server.Xenoarchaeology.XenoArtifacts.Triggers.Components;
+
+namespace Content.Server.Speech.EntitySystems;
+
+public sealed class VulgarAccentSystem : EntitySystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly ReplacementAccentSystem _replacement = default!;
+
+    private string[] _swearWords =
+    {
+        "FUCK",
+        "FUCKING HELL",
+        "GOD DAMN",
+        "SHIT",
+        "BIG SACK OF SHIT",
+        "SON OF A BITCH",
+        "BITCH",
+        "COCK",
+        "COCKSUCKER",
+        "BONER SHIT COCKS",
+        "MOTHERFUCKER",
+        "YOU BASTARD",
+        "DICK",
+        "DICKBAG",
+        "ASSHOLE",
+        "SHITTY ASS FUCKHOLE",
+        "GOOD HEAVENS",
+        "SON OF A CLUWNE"
+    };
```

Comment:
very bad to keep strings right in code, should be localizeable

Reply:
I'm on it